### ISSUE TITLE
Security is not working properly with multiple GroupedOpenApi 

### ISSUE THREAD

**Issue by RuslanKharevych** [springdoc/springdoc-openapi#915]
I want to have different security headers for Internal and Public API. But only one security schema appears on UI.
For example:

- If i open Public API page first after application startup, then only "Authorization" header can be set on both pages
- If i open Internal API page first after application startup, then only "apiKey" header can be set

Expected behavior:

- Internal API page - "apiKey"
- Public API page - "Authorization"

Dependencies:
```
<dependency>
            <groupId>org.springdoc</groupId>
            <artifactId>springdoc-openapi-ui</artifactId>
            <version>1.4.4</version>
        </dependency>

        <dependency>
            <groupId>org.springdoc</groupId>
            <artifactId>springdoc-openapi-security</artifactId>
            <version>1.4.4</version>
        </dependency>
````

Code:
```
@Configuration
@ConditionalOnProperty(name = "springdoc.api-docs.enabled", havingValue = "true")
public class SwaggerConfig {

    public SwaggerConfig() {
        SpringDocUtils.getConfig().replaceWithClass(org.springframework.data.domain.Pageable.class, Pageable.class);
    }

    @Bean
    public GroupedOpenApi publicApi() {
        return GroupedOpenApi.builder()
                .group("1.Public")
                .pathsToMatch("/**")
                .pathsToExclude("/internal/**")
                .packagesToScan("com.app.project")
                .addOpenApiCustomiser(publicApiCustomizer())
                .build();
    }

    @Bean
    public GroupedOpenApi internalApi() {
        return GroupedOpenApi.builder()
                .group("2.Internal")
                .pathsToMatch("/internal/**")
                .packagesToScan("com.app.project")
                .addOpenApiCustomiser(internalApiCustomizer())
                .build();
    }

    @Bean
    public OpenApiCustomiser publicApiCustomizer() {
        return openApi -> openApi.addSecurityItem(new SecurityRequirement().addList("Authorization"))
                .components(new Components()
                        .addSecuritySchemes("Authorization", new SecurityScheme()
                                .in(SecurityScheme.In.HEADER)
                                .type(SecurityScheme.Type.HTTP)
                                .scheme("bearer")
                                .bearerFormat("JWT"))
                )
                .info(new Info().title("Public REST API"));
    }

    @Bean
    public OpenApiCustomiser internalApiCustomizer() {
        return openApi -> openApi
                .addSecurityItem(new SecurityRequirement().addList("apiKey"))
                .components(new Components()
                        .addSecuritySchemes("apiKey", new SecurityScheme()
                                .type(SecurityScheme.Type.APIKEY)
                                .in(SecurityScheme.In.HEADER)
                                .name("apiKey")))
                .info(new Info().title("Internal REST API"));
    }

}
```

Public API authorization:
![image](https://user-images.githubusercontent.com/31544614/97697340-8b15e900-1aaf-11eb-8633-be5c16f7adac.png)

Internal API authorization(should be "apiKey"):
![image](https://user-images.githubusercontent.com/31544614/97697383-9b2dc880-1aaf-11eb-8ea9-6ccb00f8af37.png)

**Comment by bnasslahsen** [comment]
@RuslanKharevych, 

 The issue you are reporting here is not related directly to springdoc-openapi project.
 You can check the generated OpenAPI spec, where you will find the correct securtiy schemes are present, but it's more a GUI issue.

 Your issue is more related to the [swagger-ui](https://github.com/swagger-api/swagger-ui). While switching, the popup seems only to show up when will refresh the page.

 And there are already some existing issues that we are following: 

- https://github.com/swagger-api/swagger-ui/issues/6537
- https://github.com/swagger-api/swagger-ui/issues/5404
- https://github.com/swagger-api/swagger-ui/issues/5540

 Once they are fixed, the new version will be added to springdoc-openapi.

**Comment by MichielDx** [comment]
In which version of springdoc-openapi-ui is this fixed? It's not working in 1.4.4
Or is there a property that I should/can set to fix this?
